discovery-provider/src/utils/challenges.py

```diff
@@ -0,0 +1,33 @@
+import json
```

Comment:
i kinda feel like this file belongs in src/challenges as well and maybe it could just be called

src/challenges/setup.py?

Reply:
agree